R/mcnemar.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "McNemar's test in R"
+---
+
+### Performing McNemar's test in R
+
+To demonstrate McNemar's test, data was used concerning the presence or absence of cold symptoms reported by the same children at age 12 and age 14. A total of 2638 participants were involved.
+
+```{r}
+#| echo: false
+#| include: false
+library(knitr)
+
+knitr::opts_chunk$set(echo = TRUE, cache = TRUE)
+
+colds <- 
+  read.csv(
+    file = "../data/colds.csv")
+```
+
+#### Using the `epibasix::mcnemar` function
```

Comment:
Add a note in here which say that the author is no longer maintaining the epibasix package and confidence intervals do not have documentation available for the method used.  Therefore, other packages may be more suitable.  See Comparison page for more detail.